R/last_connection.R

```diff
@@ -10,11 +10,20 @@
 #' @seealso \code{\link{postgresql_connection}}
 #' @export
 last_connection <- function() {
-  get_cache('last_connection') %||%
+  conn <- get_cache('last_connection')
+  if (is(conn, "error")) {
+    conn <- NULL
+  }
+  if (is(conn, 'PqConnection')) {
+    isValidConnection <- tryCatch(dbIsValid(dbSendQuery(conn, 'select 1')), error = function(e) FALSE)
```

Comment:
I remember having issues with dbSendQuery and not fetching the result in RPostgres, hence I prefer dbGetQuery. But if this works and is tested - :+1: